src/lib/core/assessment.ts

```diff
@@ -1008,3 +1008,48 @@ export const getAssessmentDetails = async ({
   })
   return assessment
 }
+
+//thissssss
+export const getCandidateAssessments = async ({userId}: {userId: string}) => {
```

Comment:
Rename getCandidateAssessments to getManyCandidateAssessment and put this function in /lib/core/candidate.ts instead of assessment.ts